.gitignore

```diff
@@ -208,3 +208,9 @@ test/fakeintake/build/*
 
 # Allow Regression Detector material to ignore excludes
 !test/regression/**
+
+# launch template + dd agent file
```

Comment:
These ignores are specific to your environment. probably best not to include them in this PR. 
Also you deleted the vscode templates - we should undelete them.